new-contract-interaction/src/agent.spec.js

```diff
@@ -31,8 +31,9 @@ const filteredAddress = `0x3${'0'.repeat(39)}`;
 
 const config = require('../agent-config.json');
 
+// grab first contract to test
 const [testContract] = Object.keys(config.contracts);
-const { address: testContractAddress } = config.contracts[testContract];
+const { address: testContractAddress } = config.contracts[testContract].newContractEOA;
```

Comment:
Line 35 will return an array of keys. `testContract` will not handle multiple contracts correctly. Also, `.newContractEOA` is referenced but doesn't exist in agent-config.json

Reply:
yea the reason I did this was to get an array of contracts and then just referencing the first one for testing purposes

Reply:
No problem then.